README.md

```diff
@@ -7,6 +7,14 @@
 * Verify that Python-3 and pip3 should be installed in your system. terminal>`pip3 -V`
 * Install pyb:  `pip3 install pybuilder`
 * Verify Pyb  installation :`pyb --version`
+```shell
+# If pyb command is not found, you will have to add pyb to path.
+# follow these instructions to add pyb to path.
+
+# for ubuntu/bash:
+> echo 'export PATH=$PATH:/home/home/.local/bin' >> ~/.bashrc 
```

Comment:
.local is a hidden folder, This could be under the control of virtual env or any utility. Verify if this won't corrupt the global environments by doing this

Reply:
updated.